Makefile

```diff
@@ -119,7 +120,7 @@ container: .container-$(DOTFILE_IMAGE) container-name
 	    -e 's|{ARG_OS}|$(OS)|g' \
 	    -e 's|{ARG_FROM}|$(BASEIMAGE)|g' \
 	    Dockerfile.in > .dockerfile-$(OS)_$(ARCH)
-	@docker build -t $(IMAGE):$(TAG) -f .dockerfile-$(OS)_$(ARCH) .
+	@docker build --build-arg HTTP_PROXY=$(HTTP_PROXY) --build-arg=$(HTTPS_PROXY) -t $(IMAGE):$(TAG) -f .dockerfile-$(OS)_$(ARCH) .
```

Comment:
Can we line-wrap this as

```
@docker build \
    --build-arg HTTP_PROXY=$(HTTP_PROXY) \
    --build-arg=$(HTTPS_PROXY) \
    -t $(IMAGE):$(TAG) \
    -f .dockerfile-$(OS)_$(ARCH) \
    .
````

Reply:
Do we really need the build-arg?  The env var will be passed in and used in the build lines.

Reply:
Yes, I double checked behind the proxy. If i'm not passing the proxy to the docker build Step 4 will fail. See logs:
`building: bin/linux_amd64/git-sync
Sending build context to Docker daemon  13.39MB
Step 1/7 : FROM alpine:3.8
 ---> 196d12cf6ab1
Step 2/7 : MAINTAINER Tim Hockin <thockin@google.com>
 ---> Using cache
 ---> 452cffc36068
Step 3/7 : ADD bin/linux_amd64/git-sync /git-sync
 ---> f7b715a845a7
Step 4/7 : RUN apk update --no-cache && apk add     ca-certificates     coreutils     git     openssh-client
 ---> Running in cacefe191798
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz
WARNING: Ignoring http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz: temporary error (try again later)
OK: 13 distinct packages available
WARNING: Ignoring http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz: temporary error (try again later)
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz
ERROR: http://dl-cdn.alpinelinux.org/alpine/v3.8/main: temporary error (try again later)
WARNING: Ignoring APKINDEX.adfa7ceb.tar.gz: No such file or directory
ERROR: http://dl-cdn.alpinelinux.org/alpine/v3.8/community: temporary error (try again later)
WARNING: Ignoring APKINDEX.efaa1f73.tar.gz: No such file or directory
ERROR: unsatisfiable constraints:
  ca-certificates (missing):
    required by: world[ca-certificates]
  coreutils (missing):
    required by: world[coreutils]
  git (missing):
    required by: world[git]
  openssh-client (missing):
    required by: world[openssh-client]`